inginious/frontend/submission_manager.py

```diff
@@ -49,11 +49,20 @@ def _job_done_callback(self, submissionid, task, result, grade, problems, tests,
         submission = self.get_submission(submissionid, False)
         submission = self.get_input_from_submission(submission)
 
+        # Get user penalty stored in 'user_hints' collection to apply in grade of submission
+        # TODO: should be done for submissions in group tasks mode
+        username = submission["username"][0]
+        penalty = 0.0
+        if grade and username:
```

Comment:
Actually the username is never empty or None, thus checking if `username` is not necessary.